winpr/include/winpr/file.h

```diff
@@ -350,6 +350,62 @@ typedef struct _HANDLE_CREATOR
 
 #endif /* _WIN32 */
 
+#ifdef _UWP
```

Comment:
I thought of this, but in this case it was simply easier to consider UWP separately since only part of the functions were missing. The replacement code in many of the cases is using the extended or more recent version of the same function. Mixing it with the non-Win32 function code would have made it quite messy. I also only ported the required functions to allow building at this point.